src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -46,6 +46,7 @@ azure-mgmt-devtestlabs==4.0.0
 azure-mgmt-dns==8.0.0
 azure-mgmt-eventgrid==9.0.0
 azure-mgmt-eventhub==4.1.0
+azure-mgmt-extendedlocation==1.0.0b2
```

Comment:
is this package for app service extended location? For storage, compute and network, we support extended location in their own package like azure-mgmt-storage, azure-mgmt-compute and azure-mgmt-network. I am curious about app service design.

Reply:
I want to be able to use these management operations so I can verify the ExtendedLocation resource before using it in App Service: https://docs.microsoft.com/en-us/python/api/azure-mgmt-extendedlocation/azure.mgmt.extendedlocation.v2021_03_15_preview.operations.customlocationsoperations?view=azure-python-preview
We have the ExtendedLocation model defined in App Service, but don't have these management operations - does storage/compute/network verify the ExtendedLocation resources before using it?

Reply:
No, we don't.